Italian/ascoltare/profumato.md

```diff
@@ -0,0 +1,3 @@
+Parto e ascolto il programma più profumato d'Italia [Tutto Esaurito!] (http://www.105.net/sezioni/651/tutto-esaurito)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.